tests/inst/src/rpmostree.rs

```diff
@@ -25,9 +24,29 @@ pub(crate) struct Deployment {
 }
 
 pub(crate) fn query_status() -> Result<Status> {
-    let cmd = Command::new("rpm-ostree")
-        .args(&["status", "--json"])
-        .stdout(Stdio::piped())
-        .spawn()?;
-    Ok(serde_json::from_reader(cmd.stdout.unwrap())?)
```

Comment:
Ugh and we were missing a check of the exit status here...such an evil trap in the Rust command API.
(Which you have correctly fixed)
I have this bit https://github.com/cgwalters/coretoolbox/blob/29a862b62587e1b462d577a45037e8c395e654ab/src/cmdrunext.rs#L8 we could cargo cult around or...perhaps crateify.  I thought https://docs.rs/subprocess/0.2.6/subprocess/ had an API for this but I guess not?